util/get_fisher_data.py

```diff
@@ -208,22 +209,50 @@ def append_special_place_names(places):
         else:
           places['de'][place] = [places['de'][place], m.group(1)]
 
-# Actual program runs here
-places, fishes, placefish = get_fish_data()
-tackle = get_tackle()
-append_special_place_names(places)
-
-data = {
-    'tackle': tackle,
-    'places': places,
-    'fish': fishes,
-    'placefish': placefish
-}
-
-filename = Path(__file__).resolve().parent.parent / 'ui' / 'fisher' / 'static-data.js'
-writer = coinach.CoinachWriter()
-writer.write(
-    filename,
-    os.path.basename(os.path.abspath(__file__)),
-    'gFisherData',
-    data)
+def get_cn_data():
+    global locales, base
+
+    locales = ['chs', 'en']
+    base = "https://cafemaker.wakingsands.com/"
+
+    places, fishes, _ = get_fish_data()
+    tackle = get_tackle()
+
+    return places, fishes, tackle
+
+
+
+if __name__ == "__main__":
+    example_usage = "python3 get_fisher_data.py"
+    parser = argparse.ArgumentParser(
+        description="Generate fisher data from xivapi",
+        epilog=example_usage,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    parser.add_argument('-cn', '--chs', help="If passed, get fisher data from cn xivapi", action='store_true')
```

Comment:
Can you make this always happen? I don't know why we would not want to get fisher data from xivapi?

Reply:
sorry for the late reply, I made this default.